test/integration/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ DUP1,4
     add_test(NAME ${PREFIX}/trace COMMAND $<TARGET_FILE:evmc::tool> --vm $<TARGET_FILE:evmone>,trace run 60006002800103)
     set_tests_properties(
         ${PREFIX}/trace PROPERTIES PASS_REGULAR_EXPRESSION
-        "{\"depth\":0,\"rev\":\"London\",\"static\":false}
+        "{\"depth\":0,\"rev\":\"Paris\",\"static\":false}
```

Comment:
Maybe merge this commit into the Prague commit and extend the commit log with "Additionally EVMC set the latest stable revision to Paris".